File: web/src/model/victron_controller/dashboard/from_0_2_0_timer.ts
// @ts-nocheck
import {Timer as v0_2_0_Timer} from './v0_2_0/Timer'
import {Timer as dashboard_Timer} from './Timer'

export function convert__timer__from__0_2_0(from: v0_2_0_Timer): dashboard_Timer {
    return new dashboard_Timer (
        from.id,
        from.description,
        from.period_ms,
        from.last_fire_epoch_ms,
        from.next_fire_epoch_ms,
        from.status
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_timer.rs


pub fn convert__timer__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::timer::Timer) -> crate::victron_controller::dashboard::timer::Timer {
    crate::victron_controller::dashboard::timer::Timer {
        id: from.id.clone(),
        description: from.description.clone(),
        period_ms: from.period_ms.clone(),
        last_fire_epoch_ms: from.last_fire_epoch_ms.clone(),
        next_fire_epoch_ms: from.next_fire_epoch_ms.clone(),
        status: from.status.clone(),
    }
}